modules/ROOT/partials/configuration/valid_elements.adoc

```diff
@@ -7,14 +7,14 @@ This option contains a comma separated list of element conversion chunks. Each c
 
 If you just want to add or change some behavior for a few items, use the xref:extended_valid_elements[`extended_valid_elements`] option
 
-[discrete]
 === Control characters
 
+[cols="1,6"]
 |===
 | Name | Summary
 
 | @
-| Rules with this name will be applied to all elements defined after this rule. So `+@[attr1&#124;attr2]+` will enable `attr1` and `attr2` for all elements, but `+element1,@[attr1&#124;attr2],element2,element3+` will enable `attr1` and `attr2` only for `element2` and `element3`. If applied in xref:extended_valid_elements[`extended_valid_elements`], it is only effective for the elements in the extended_valid_elements list.
+| Rules with this name will be applied to all elements defined after this rule. So `+@[attr1\|attr2]+` will enable `attr1` and `attr2` for all elements, but `+element1,@[attr1\|attr2],element2,element3+` will enable `attr1` and `attr2` only for `element2` and `element3`. If applied in xref:extended_valid_elements[`extended_valid_elements`], it is only effective for the elements in the extended_valid_elements list.
```

Comment:
There is a ref in here that refers to another file, it's better to include the full ref:
```suggestion
| Rules with this name will be applied to all elements defined after this rule. So `+@[attr1\|attr2]+` will enable `attr1` and `attr2` for all elements, but `+element1,@[attr1\|attr2],element2,element3+` will enable `attr1` and `attr2` only for `element2` and `element3`. If applied in xref:configure/content-filtering.adoc#extended_valid_elements[`extended_valid_elements`], it is only effective for the elements in the extended_valid_elements list.
```